### ISSUE TITLE
Does v1.4 model respect punctuation?

### ISSUE THREAD

**Issue by hoveychen** [fishaudio/fish-speech#542]
### Self Checks

- [X] I have searched for existing issues [search for existing issues]([https://github.com/langgenius/dify/issues](https://github.com/fishaudio/fish-speech/issues)), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/fishaudio/fish-speech/issues/515)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

We've tried synthesizing voices in both English and Chinese dialog scripts, and it seems to ignore the punctuation like question mark.

Example: 
* 他为什么要劈腿？我到底哪里不好？
* 我男朋友在和别的女人在一起，真人秀？
* Do you hear the whispers of the planets, the gentle hum of the galaxies far beyond our reach? 

### 3. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by czkoko** [comment]
I found that if there is a semicolon `;` in the sentence, the whole result will become a noise.

**Comment by Stardust-minus** [comment]
The training data has been processed by punctuation normalizer, so you should normalize the normalizer in inference.

**Comment by leng-yue** [comment]
Not sure why, but technically puncs are normalized here: https://github.com/fishaudio/fish-speech/blob/c7c8c943c966a03a85ce4a61bca605f1d9bf7567/fish_speech/text/clean.py#L28
We are finetuning the model to let it understand different puncs.

**Comment by PoTaTo-Mika** [comment]
Closed as issue had been solved.
